als-suggestions/shared/src/main/scala/org/mulesoft/als/suggestions/aml/VocabularyDialectPluginRegistry.scala

```diff
@@ -4,18 +4,17 @@ import amf.aml.client.scala.model.document.Dialect
 import org.mulesoft.als.suggestions.AMLBaseCompletionPlugins
 import org.mulesoft.als.suggestions.aml.webapi.WebApiCompletionPluginRegistry
 import org.mulesoft.als.suggestions.interfaces.AMLCompletionPlugin
-import org.mulesoft.als.suggestions.plugins.aml.vocabulary.{
-  ClassTermUriCompletionPlugin,
-  PropertyTermUriCompletionPlugin
-}
+import org.mulesoft.als.suggestions.plugins.aml.vocabulary.{ClassTermUriCompletionPlugin, PropertyTermUriCompletionPlugin}
+import org.mulesoft.als.suggestions.plugins.aml.webapi.raml.AMLLibraryPathCompletion
 import org.mulesoft.amfintegration.dialect.dialects.metadialect.VocabularyDialect
 
 object VocabularyDialectPluginRegistry extends WebApiCompletionPluginRegistry {
 
   private val all: Seq[AMLCompletionPlugin] =
     AMLBaseCompletionPlugins.all :+
       PropertyTermUriCompletionPlugin :+
-      ClassTermUriCompletionPlugin
+      ClassTermUriCompletionPlugin :+
+      AMLLibraryPathCompletion
```

Comment:
ahy are you adding the pluygin here?

Reply:
Vocabularies use a similar way of importing modules as RAML

Reply:
Porque al sacarlo del object AMLBaseCompletionPlugins ya no se esta agregando en el VocabularyDialectPluginRegistry y por los test entiendo que debe seguir estando. Es correcto?